crates/superposition_types/Cargo.toml

```
@@ -22,6 +22,7 @@ log = { workspace = true }
 regex = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
+strum = { workspace = true }
```

also, need to add it in deps for `experimentation` feature here
```suggestion
strum = { workspace = true, optional = true }
```